assignment1/src/main/java/ro/utcn/sd/he/assignment1/model/Vote.java

```diff
@@ -4,13 +4,21 @@
 import lombok.Data;
 import lombok.NoArgsConstructor;
 
+import javax.persistence.*;
+
 @Data
 @NoArgsConstructor
 @AllArgsConstructor
+@Entity
 public class Vote {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
     private int id;
     private int type;
+    @Column(name = "userID")
     private int userID;
-    private int questionID;
-    private int answerID;
+    @Column(name = "questionID")
+    private Integer questionID;
```

Comment:
Meh, you modeled all your Hibernate relationships by just mapping the foreign keys (so you have no `@OneToMany` or `@ManyToMany`). I assume that this is because you implemented this first for JDBC (and then it would have been too much effort to move to a "better" model for Hibernate).

Reply:
Yes, I should've started with hibernate to begin with. Next time I'll know.